pages/docs/reference/using-gradle.md

```diff
@@ -313,12 +311,27 @@ plugins {
 <div class="sample" markdown="1" mode="kotlin" theme="idea" data-lang="kotlin" data-highlight-only>
 
 ```kotlin
-buildscript {
-    dependencies {
-        classpath("com.android.tools.build:gradle:3.2.1")
-        classpath(kotlin("gradle-plugin", version = "{{ site.data.releases.latest.version }}"))
+// settings.gradle.kts
+pluginManagement {
+    repositories {
+        gradlePluginPortal()
+        google()
+    }
+
+    resolutionStrategy {
+        eachPlugin {
+            if (requested.id.id in setOf("kotlin-android", "kotlin-kapt", "kotlin-android-extensions")) {
+                useModule("org.jetbrains.kotlin:kotlin-gradle-plugin:${requested.version}")
+            }
+
+            if (requested.id.id in setOf("com.android.application", "com.android.feature")) {
```

Comment:
Why not to use `if(requested.id.namespace == "com.android") {` as in the Gradle example here: https://docs.gradle.org/current/userguide/kotlin_dsl.html#sec:plugins_resolution_strategy ?
And similar for Kotlin plugins.

Reply:
That would also work. I did this back when this wasn't documented and had to dig through source code, so there is no reason other than this is what I saw gradle doing at that time.

Reply:
Why such a complex solution with custom resolutionStrategy?

In Kotlin normally works just
```
    kotlin("android") version("1.3.72")
    kotlin("android.extensions") version("1.3.72")
```

And in Groovy
```
    id "org.jetbrains.kotlin.android" version('1.3.72')
    id "org.jetbrains.kotlin.android.extensions" version('1.3.72')

```

Reply:
This is for loading the Kotlin extensions it's for loading the android plugin which isn't available in the Gradle plugins portal, and I don't believe the kotlin ones will load the way you suggested either due to how the ids and plugins are named.

Reply:
You have right, this works for me
```
plugins {
    id 'com.android.application'
    id "org.jetbrains.kotlin.android"
}

```must because I have this in the root gradle file

```
buildscript {
    ext.kotlin_version = "1.3.72"
    repositories {
        google()
        jcenter()
    }
    dependencies {
        classpath "com.android.tools.build:gradle:4.1.0-beta03"
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
    }
}

```